app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -52,16 +58,16 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
             app:shadow_radius="8"
-            app:srcCompat="@{@drawable/ic_add_profile}"/>
+            app:srcCompat="@{@drawable/ic_default_avatar}"/>
```

Comment:
Was this conflict resolved correctly? This change seems unrelated to your PR.

Reply:
I was trying to resolve the merge conflict, however I've now reverted the changes back to original

Reply:
It seems that you've reverted this change after the reformat. Please fix.

Also, it may help to double check your code before sending it back for review to make sure small stuff like this & the reformatting can be caught earlier rather than needing for a reviewer to bring it up.

Reply:
Hi @BenHenning , the changes have been reverted. PTAL